draft-irtf-cfrg-hpke.md

```diff
@@ -554,6 +554,42 @@ def Context.Open(aad, ct):
   return pt
 ~~~~~
 
+# Single-Shot APIs
+
+In many cases, applications encrypt only a single message to a recipient's public key. 
+This section provides templates for HPKE APIs that implement "single-shot" encryption 
+and decryption using APIs specified in {{hpke-kem}} and {{hpke-dem}}. Without loss
+of generality, assume the following function templates exist for single-shot encryption
+and decryption, respectively:
+
+~~~~~
+def Seal<MODE>(pkR, info, aad, pt, ...):
+  enc, ctx = SetupI<MODE>(pkR, info, ...)
+  ct = ctx.Seal(aad, pt)
+  return enc, ct
+
+def Open<MODE>(skR, info, enc, aad, ct, ...):
+  ctx = SetupR<MODE>(enc, skR, info, ...)
+  return ctx.Open(aad, ct)
+~~~~~
+
+The `MODE` template parameter is one of Base, PSK, Auth, or AuthPSK. Optional parameters
+to SetupI<MODE> and SetupR<MODE> depend on the `MODE` and may be empty. SetupBase, for example, 
+has no additional parameters. 
+
+Using such a template, SetupAuthPSK and OpenAuthPSK might be implemented as follows:
```

Comment:
Don't you mean `SealAuthPSK`?

Reply:
s/Using such a template/Thus/
s/might be/would be/

Reply:
> Don't you mean SealAuthPSK?
 
Oops, yeah. Will fix.